requirements/minimal_requirements.txt

```diff
@@ -0,0 +1,40 @@
+# we use this file to track minimal dependencies
+certifi==2017.4.17
+hydra-core==1.0.0
+lightly_utils~=0.0.0
+numpy==1.21.6
+python_dateutil==2.5.3
+requests==2.23.0
+six==1.10
+tqdm==4.57.0
+urllib3==1.25.3
+pydantic==1.10.5
+aenum==3.1.11
+pytorch_lightning==1.7.1
+torch==1.11.0
+torchvision==0.12.0
```

Comment:
Could you put these as minimal version in `requirements/torch.txt`?

Reply:
I thought about this but decided against it. There are still other modules that can be used without the latest PyTorch and torchvision packages. E.g. if someone just cares about a loss function from lightly but uses still older PyTorch or Torchvision packages, we would force them to upgrade.

On the torchvision side, many dependency constraints come from the video processing (a feature not many users use).